airflow/www/views.py

```diff
@@ -4534,6 +4535,14 @@ def action_set_retry(self, tis):
         self.update_redirect()
         return redirect(self.get_redirect())
 
+    @action('set_skipped', "Set state to 'skipped'", '', single=False)
+    @action_has_dag_edit_access
+    def action_set_skipped(self, tis):
+        """Set state to skipped."""
+        self.set_task_instance_state(tis, State.SKIPPED)
```

Comment:
```suggestion
        self.set_task_instance_state(tis, TaskInstanceState.SKIPPED)
```

We’re actively adding type hints to the code base and this enum is more typing-friendly.

Reply:
When I ran this locally `TaskInstanceState` was undefined. Does it need to be imported?